dry_rest_permissions/generics.py

```diff
@@ -188,6 +224,13 @@ class DRYObjectPermissions(DRYPermissions):
     global_permissions = False
 
 
+class DRYOrDjangoPermissions(DRYPermissions):
+    """
+    This is a shortcut class that can be used to check django permissions if permission methods aren't defined.
+    """
```

Comment:
Ok for the class but maybe a bit more docs to explain for a person that don't use it often and also a doc in the README